modules/imgcodecs/perf/perf_jpeg.cpp

```diff
@@ -0,0 +1,34 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html
+#include "perf_precomp.hpp"
+
+namespace opencv_test
+{
+    using namespace perf;
+
+    typedef perf::TestBaseWithParam<std::string> JPEG;
+
+    PERF_TEST(JPEG, Decode)
+    {
+        String filename = getDataPath("stitching/boat1.jpg");
+
+        TEST_CYCLE() imread(filename);
```

Comment:
It makes sense to benchmark imdecode and imencode. They do not read from /write to disk and benchmarks shows pure time.